.gitignore

```diff
@@ -1,4 +1,5 @@
 .envrc
+.vscode/
```

Comment:
I was asked not to include IDE specific entries to gitignore.
You can add them to you local git ignore like this: https://stackoverflow.com/a/1753078

Reply:
Projects like kubernetes https://github.com/kubernetes/kubernetes/blob/master/.gitignore and opentofu https://github.com/opentofu/opentofu/blob/main/.gitignore are including them, I'll add them to my local one for now

Reply:
removed!